system/html.php

```diff
@@ -560,15 +560,15 @@ public static function multiColForm($data, $action = null, $method = "POST", $su
             foreach ($rows as $row) {
                 
                 // Print each field
-                $fieldCount = count($row);
+                $fieldCount = !empty($row) ? count(array_filter($row)) : 1;
```

Comment:
What is the purpose of array_filter here? Read up on what it actually does: http://php.net/manual/en/function.array-filter.php